fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -195,6 +197,31 @@
         tripleFormat.setTimeZone(getTimeZone("GMT"));
     }
 
+    private static File TEST_FILE;
```

Comment:
I believe only big size of file can trigger the SN-sibling bug as reported in https://jira.duraspace.org/browse/FCREPO-2055, smaller file will trigger other errors that even FedoraLdp rest api fails to handle.